src/base/system_parameter.h

```diff
@@ -600,6 +601,8 @@ extern "C"
 					 * this flag only can be set if the parameter has PRM_FOR_CLIENT,
 					 * PRM_FOR_SERVER, and PRM_USER_CHANGE flags. */
 
+#define PRM_FOR_PL_CONTEXT  0x00010000	/* is for PL parameter */
```

Comment:
```suggestion
#define PRM_FOR_PL_CONTEXT  0x00020000	/* is for PL parameter */
```
@vimkim Thank you very much for your careful review